docker/Dockerfile

```diff
@@ -33,36 +33,34 @@ RUN bash ./Miniconda-latest-Linux-x86_64.sh -b
 RUN git clone https://github.com/memex-explorer/memex-explorer
 
 WORKDIR /home/explorer/memex-explorer/
-RUN git checkout origin/ahmadia/tad_integration_master
 
 WORKDIR /home/explorer/memex-explorer
-ENV PATH /home/explorer/miniconda/bin:$PATH
+ENV PATH /home/explorer/miniconda2/bin:$PATH
```

Comment:
I couldn't find this changed documented anywhere, but I strongly suspect it's new.